src/Components/dashboard/route/AddRouteButton.jsx

```diff
@@ -29,10 +29,17 @@ const AddRouteButton = () => {
     const [endPointError, setEndPointError] = useState(false);
     const [stopError, setStopError] = useState(false);
 
+    const [openUpdater, setOpenUpdater] = useState(false);
+
     const handleClear = () => {
         console.log("Content cleared");
```

Comment:
I think you should use this function to clear all states you have used while getting values from inputs instead of  putting `console.log("Content cleared");` and then call it after submitting the values

Reply:
We don't need this here I will remove it as we created already an update component!